app/src/main/java/org/simple/clinic/activity/TheActivityComponent.kt

```diff
@@ -103,6 +104,7 @@ interface TheActivityComponent {
   fun inject(target: QrCodeScannerView)
   fun inject(target: RecentPatientsView)
   fun inject(patientsScreenKey: PatientsScreenKey)
+  fun inject(target: SyncIndicatorView)
```

Comment:
sidenote: @rakshakhegde can you rename `patientsScreenKey` to `target`? Asking you because this looks like a part of your feature.